test/sharness/t0276-cidv0v1.sh

```diff
@@ -28,8 +28,8 @@ test_expect_success "add file using CIDv0" '
 '
 
 test_expect_success "check hashes" '
-  test "$(cid-fmt %v-%c $AHASHv0)" = "cidv0-protobuf" &&
```

Comment:
why did you need to change this?

Reply:
Because I updated `go-cid` to `v0.2.0` inside tool's `go.mod`.
Which has `dag-pb` instead.